DataStructures.Tests/UnitTest1.cs

```diff
@@ -25,13 +25,13 @@ public void Insert_adds_value_to_start()
             list.Equals(1);
 
             // Assert
-            Assert.Equal("{ 1 } -> NULL", list.ToString());

```

Comment:
What you deleted is what you're going to want when you do end up writing the logic for this to work properly.